pipeline.lisp

```diff
@@ -83,21 +122,26 @@ segments. You should always use remove-segment instead if possible.")
         (vector-pop array))))
 
 (defgeneric add-segment (pipeline segment &optional place)
-  (:documentation "Add a new segment to the pipeline.
-If place is set, the pipe is added to the specified place as per INSERT.
-The place specified is expected to be a splitter or similar to append
-the pipe to. Returns the segment.")
+  (:documentation "Add a new segment to the pipeline.  If place is set, the pipe is added
```

Comment:
Why was this reformatted? The first line of a docstring should always be a single-sentence short description.

Reply:
Likely because I went back and forth on what the API should be, and then I missed that the docstring got reformatted even though I reverted the changes to the code.